packages/gatsby-plugin-typography/src/__mocks__/typography-plugin-cache-endpoint.js

```diff
@@ -0,0 +1,2 @@
+import Typography from "typography"
+export default new Typography()
```

Comment:
do we need __mocks__ file if we are doing `doMock` for it anyway in tests?

Reply:
You're right; I had to go back and look again, because for some reason I was under the impression that it was using `require('typography-plugin-cache-endpoint')`, but it makes much more sense that it use `js.mock()`/`js.doMock()`, so I'm not sure why I was thinking that.  Perhaps I was transposing the way it is used in `gatsby-browser` onto the tests.

Thanks again.  Deleted.